jvm-packages/xgboost4j-gpu/src/test/java/ml/dmlc/xgboost4j/gpu/java/BoosterTest.java

```diff
@@ -83,18 +83,19 @@ public void testBooster() throws XGBoostError {
 
     try (Table tmpTable = Table.readCSV(schema, opts,
 					new File(trainingDataPath))) {
-      CudfDataFrame cudfDataFrame = new CudfDataFrame(tmpTable, new int[]{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11}, new int[]{12});
+      CudfColumnBatch batch = new CudfColumnBatch(tmpTable, new int[]{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11}, new int[]{12});
+      CudfColumn labelColumn = CudfColumn.from(tmpTable.getColumn(12));
```

Comment:
> , new int[]{12}

These label indices should not be here right?

Reply:
the batch will be used to create the Normal DMatrix which does not require label column and the DeviceDmatrix which require label column